rules/endpoints.go

```diff
@@ -293,6 +293,19 @@ func (r *DefaultRuleRenderer) endpointIptablesChain(
 		},
 	})
 
+	rules = append(rules, Rule{
+		Match: Match().ProtocolNum(ProtoUDP).
+			DestPorts(uint16(r.Config.VXLANPort)).
+			VXLANVNI(uint32(r.Config.VXLANVNI)),
+		Action:  DropAction{},
+		Comment: "Drop VXLAN encapped packets originating in pods",
+	})
+	rules = append(rules, Rule{
+		Match:   Match().ProtocolNum(ProtoIPIP),
+		Action:  DropAction{},
+		Comment: "Drop IPinIP encapped packets originating in pods",
+	})
+
```

Comment:
I believe these rules will be enforced both (1) on egress from a local workload, and (2) on ingress **to** a local workload.  Right?  I understand that we definitely want (1), but do we really want to enforce (2) as well?

Reply:
What about when we go through these rules for a packet from or to a host endpoint?  Do we want to enforce them in that case?

Reply:
(Relatedly, do we want these rules in raw and pre-DNAT endpoint chains?  raw and pre-DNAT are used for host endpoints only, but the pre-DNAT now allows the "*" host endpoint which includes local workload interfaces as well...)

Reply:
Fixed per our conversation on slack